src/commands.ts

```diff
@@ -223,8 +223,14 @@ export const inputHandlers = (store: Store<State, any>) => ({
 
     // Get the command from the command gesture index.
     // When the command palette  is displayed, disable gesture aliases (i.e. gestures hidden from instructions). This is because the gesture hints are meant only as an aid when entering gestures quickly.
-    const command =
-      !state.showCommandPalette || !commandGestureIndex[sequence as string]?.hideFromHelp
+    const helpCommand = commandIdIndex['help']
```

Comment:
Let's use `commandById` rather than accessing `commandIdIndex` directly. We already have `commandById`, so we might as well use and it and hide `commandIdIndex` as an implementation detail.

https://github.com/cybersemics/em/blob/75b8c516e70922bdfa44d29a8cc47e33c942e865/src/commands.ts#L325-L326